tests/lib/rules/jsx-indent-props.js

```diff
@@ -195,6 +195,48 @@ ruleTester.run('jsx-indent-props', rule, {
         },
       ],
     },
+    {
+      code: `
+       const F = () => {
+         const foo = true
+          ? <div id="id">test</div>
+          : false;
+
+          return <div
+            id="id"
+          >
+            test
+          </div>
```

Comment:
```suggestion
        return <div
          id="id"
        >
          test
        </div>
```
the return statement should always be at the same indentation level as the `const`

Reply:
You are right - somehow I've screwed up the indents for the whole code within the template string. I've amended the fix.